bolt/src/main/java/com/slack/api/bolt/App.java

```diff
@@ -423,16 +449,24 @@ public App(AppConfig appConfig, Slack slack, List<Middleware> middlewareList) {
         this.middlewareList = middlewareList;
 
         this.oAuthStateService = new ClientOnlyOAuthStateService();
+        this.openIDConnectNonceService = new NullOpenIDConnectNonceService();
 
         this.installationService = new FileInstallationService(this.appConfig);
         this.tokensRevokedEventHandler = new DefaultTokensRevokedEventHandler(this.installationService, this.executorService);
         this.appUninstalledEventHandler = new DefaultAppUninstalledEventHandler(this.installationService, this.executorService);
         this.oAuthSuccessHandler = new OAuthDefaultSuccessHandler(this.appConfig, this.installationService);
         this.oAuthV2SuccessHandler = new OAuthV2DefaultSuccessHandler(config(), this.installationService);
+        this.openIDConnectSuccessHandler = (request, response, apiResponse) -> {
+            log.warn("This OpenIDConnectSuccessHandler does nothing. " +
+                    "Implement your own handler and register it " +
+                    "by calling App#openIDConenctSuccess(handler)");
```

Comment:
```suggestion
                    "by calling App#openIDConnectSuccess(handler)");
```

Also, should we provide hints as to how to instantiate `handler`? I.e. instantiate a new `com.slack.api.bolt.service.builtin.oauth.OpenIDConnectSuccessHandler`? Or perhaps there is a document we can link to? I couldn't find any relevant documentation at https://slack.dev/java-slack-sdk/guides/web-api-basics on this topic

Reply:
For document, we can add a new page that is similar to the OAuth guide: https://slack.dev/java-slack-sdk/guides/app-distribution 

Also, javadocs are available for learning what interface/methods are available without reading source code: https://oss.sonatype.org/service/local/repositories/releases/archive/com/slack/api/bolt/1.9.0/bolt-1.9.0-javadoc.jar/!/com/slack/api/bolt/service/builtin/oauth/OAuthV2SuccessHandler.html
<img width="600" src="https://user-images.githubusercontent.com/19658/128098214-520b541d-6556-409e-b4c7-679887974a8b.png">

Reply:
Created a task issue for document: https://github.com/slackapi/java-slack-sdk/issues/799